packages/schematics/src/collection/ngrx/files/__directory__/__fileName__.reducer.ts__tmpl__

```diff
@@ -0,0 +1,24 @@
+import { <%= className %>, <%= className %>State } from './<%= fileName %>.interfaces';
```

Comment:
I thought we decided to remove interfaces to be closer to the default ngrx schematics.

Did they add the .interfaces file?